tests/ui/manual_ilog2.fixed

```diff
@@ -0,0 +1,12 @@
+#![warn(clippy::manual_ilog2)]
+
+fn main() {
+    let a: u32 = 5;
+    let _ = a.ilog2();
+    let _ = a.ilog2();
+
+    let b: u64 = 543534;
+    let _ = b.ilog2();
+
+    let _ = 64 - b.leading_zeros(); // No lint
```

Comment:
Is it check `// No lint`

Suppose no need to check this additionally

Reply:
The manual ilog2 implementation is `bit_width - 1 - x.leading_zeros()` so this is just  that any other constant besides `bit_width - 1` does not lint.